### PR TITLE
add rgb_cameras as a parameter, can be set to false to properly publish mono camera images  

### PR DESCRIPTION
**Pull request by skpawar1305**
…otos/bosdyn/api/proto_reference.html?highlight=pixelformat#image-pixelformat work

allow to not set it with a parameter

### CODE REVIEW

File: spot_driver/spot_driver/spot_ros2.py
Comment (jbarry-bdai) [inline spot_driver/spot_driver/spot_ros2.py:1109]: There should be a corresponding submodule change for `SpotWrapper` with this I think?

Comment (skpawar1305) [reply spot_driver/spot_driver/spot_ros2.py:1109]: sorry for not mentioning here..

https://github.com/bdaiinstitute/spot_wrapper/pull/15

Comment (jbarry-bdai) [reply spot_driver/spot_driver/spot_ros2.py:1109]: Right but you also need to change the submodule pointer in this repository to point to the updated version in `spot_wrapper`.

**Comment by jbarry-bdai** [comment]
@DavidWatkins can you take a look at this one when you get a chance?

**Comment by davidwatkins-bdai** [comment]
I was about to test this, however I would wait until #72 is in, merge those changes, and then we can test this.

**Comment by jbarry-bdai** [comment]
#72 is in.  do we want to do further testing onsite or ask @skpawar1305 to do more testing?

**Comment by davidwatkins-bdai** [comment]
I'll take care of testing this today.

**Comment by jbarry-bdai** [comment]
@bhung-bdai @mpickett-bdai can one of you run this PR and make sure images are still publishing?  if you verify that we can approve and push it.

**Comment by mpickett-bdai** [comment]
@jbarry-bdai Also looks good to go. Once one of these two camera PRs is fully merged we can do a test with the other to make sure the work well together if needed.

**Review by jbarry-bdai — APPROVED** [review]

**Pull request merged**
